core/modules/cart/types/ShippingMethodsData.ts

```diff
@@ -0,0 +1,15 @@
+export interface ShippingInfoAddress {
+  firstname: string,
+  lastname: string,
+  city: string,
+  postcode: string,
+  street: string[]
+}
+
+export interface ShippingMethodsData {
```

Comment:
```suggestion
export interface OrderShippingDetails {
```
or something a bit more self explaining